src/gen_server2.erl

```diff
@@ -1275,7 +1275,7 @@ name_to_pid(Name) ->
         undefined ->
             case whereis_name(Name) of
                 undefined ->
-                    exit(could_not_find_registerd_name);
+                    exit(could_not_find_registered_name);
```

Comment:
I'm not sure what erlang is doing here, so this change might be incorrect...

Reply:
It's just a reason the process terminates with. I doubt it's a common scenario, there are two call sites that use this atom across all repos (both times in `rabbitmq_common` while exiting, enver handling exits).

Let's leave this in.

Reply:
I can't tell what you mean by "leave this in" -- you could mean "leave the existing code in" or "leave the proposed replacement code in"

Reply:
Leave the proposed replacement.